list_port_forwards/display_port_forwards.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo "__(.)= __(.)=    =(.)__ =(.)__"
+echo "\___)  \___)      (___/  (___/"
+
+echo "---- Port forwards in use ----"
+
+jqquery='.items[] | { name:.metadata.annotations."tilt.dev/resource", p:.spec.forwards[] } | "\(.name):\(.p.containerPort) -> \(.p.host):\(.p.localPort)"'
+
+startquote='s/^"//'
+endquote='s/"$//'
+
+tilt get portforwards -o json | jq "$jqquery" | sed -e "$startquote" -e "$endquote"
```

Comment:
This assumes the presnce of `jq` on the user's machine...

Reply:
ya, maybe add a README that mentions that this extension depends on `jq`?

also, add a check at the top like

```
command -v jq || (echo "jq not installed :(" && exit 1)
```

Reply:
Ah, thank you for pointing that out -- I've added the check.